Formula/c/cling.rb

```diff
@@ -8,6 +8,11 @@ class Cling < Formula
     { "Apache-2.0" => { with: "LLVM-exception" } }, # llvm
   ]
 
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)$/i)
+  end
```

Comment:
```suggestion
  # There can be a notable gap between when a version is tagged and a
  # corresponding release is created, so we check the "latest" release instead
  # of the Git tags.
  livecheck do
    url :stable
    strategy :github_latest
  end
```

The previous `livecheck` block used `GithubLatest` before it was removed. v1.0 was tagged on 2023-12-09 and the release wasn't until 2024-01-28, so it seems like we still need this.